x/tally/keeper/endblock.go

```diff
@@ -109,31 +121,27 @@ func (k Keeper) ProcessTallies(ctx sdk.Context, coreContract sdk.AccAddress) err
 
 			k.Logger(ctx).Info("completed tally", "request_id", req.ID)
 			k.Logger(ctx).Debug("tally result", "request_id", req.ID, "tally_result", tallyResults[i])
+
+			// TODO
+			distMsgs = types.DistributionMessages{
+				Messages:   []types.DistributionMessage{},
+				RefundType: types.DistributionTypeNoConsensus,
```

Comment:
Maybe we should add one for insufficient commits/reveals :x

Reply:
Not sure what you mean here? :) The switch statement covers those cases right?

Reply:
Your switch case does! I was thinking about the `RefundType` message. It would be more specific for the end user than saying `No Consensus`.

Reply:
Idk what do you think @Thomasvdam?

Reply:
Ah yes I find the current list of distribution types to be confusing.
```
const (
	DistributionTypeTallyReward     DistributionType = "tally_reward"
	DistributionTypeExecutorReward  DistributionType = "executor_reward"
	DistributionTypeTimedOut        DistributionType = "timed_out"
	DistributionTypeNoConsensus     DistributionType = "no_consensus"
	DistributionTypeRemainderRefund DistributionType = "remainder_refund"
)
```
We have to add executor reward can be inferred from the distribution kind. Also, time out and no consensus are outcomes of tally, not distribution types, right? We also have to add distribution kinds for slashing and data provider payout, but I'm not sure what distribution types they would be.

Reply:
As for data provider payout I would assume it would be a `DistributionSend` message, while I thought we weren't doing any slashing to executors?